openslides_backend/models/checker.py

```diff
@@ -180,11 +180,16 @@ def __init__(
         self.models: Dict[str, Type["Model"]] = {
             collection.collection: model_registry[collection]
             for collection in model_registry
+            if not collection.collection.startswith("fake_model")
```

Comment:
We should not add test code to the "real" code. What is the exact issue why you think you need this? If it's really necessary, you should change the tests where the fake models are added so that they are removed again afterwards.

Reply:
We need to clean the model_registry of the other tests then, because some tests add fake_models to the registry and that leads to errors in this tests.

Reply:
Add code to the setup of the tests, which cleans the registry.